src/pages/[...rest].ts

```diff
@@ -156,16 +155,26 @@ async function validateRef(name: string) {
 }
 
 type Platform = typeof PLATFORMS extends Set<infer T> ? T : never;
-const PLATFORMS = new Set(['idx', 'stackblitz', 'codesandbox', 'github', 'gitpod'] as const);
+const PLATFORMS = new Set(['firebase-studio', 'stackblitz', 'codesandbox', 'github'] as const);
+const FALLBACKS = new Map<string, Platform>([['idx', 'firebase-studio']]);
```

Comment:
Nit but do we really need the affordances of a `Map` here? Using a regular object makes the code easier to read vs the nested tuples definition:

```suggestion
const FALLBACKS: Record<string, Platform> = {
  idx: 'firebase-studio',
};
```